docs/versions.md

```diff
@@ -3,5 +3,6 @@ The table below is showing all released SSE R-plugin versions and which SSE prot
 
 | __R-plugin Version__ | __SSE Protocol Version__ | __Qlik Sense Version__ | __QlikView Version__ |
 | ----- | ----- | ----- | ----- |
+| [v1.2.0](https://github.com/qlik-oss/sse-r-plugin/releases/tag/v1.2.0) | [v1.1.0](https://github.com/qlik-oss/server-side-extension/releases/tag/v1.1.0) | Qlik Sense Feb 2018 release | QlikView does not use this yet |
```

Comment:
Just a matter of preferences :) , but I would say SSE v1.1.0 is not supported in QV (or similar), rather than "..does not use this yet."

Reply:
correct, I was thinking about that even if QlikView is not yet using SSE v1.1.0 yet it is still possible for QV to use and connect to a plugin that is supporting v1.1.0 . But I think I change the text here and put the "backwards compatibility" description in the release notes instead.